src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
+  width: 1200px;
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --active-color: #00acdc;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+}
+
+.navigation {
+  justify-content: center;
+  margin-right: 40px;
+}
+
+.logo {
+  margin-left: 50px;
+}
```

Comment:
do not use margin left and right like this. better set `padding: 0 50px;` for navigation or header